Shared/Samples/Add raster from service/AddRasterFromServiceView.swift

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddRasterFromServiceView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// A Boolean value indicating whether the map is drawing.
+    @State private var mapIsDrawing = false
```

Comment:
Why not just use the `drawStatus` directly? So this property could be:
```swift
@State private var currentDrawStatus: DrawStatus = .inProgress
```

Then you can change the code in the draw status closure to:
```swift
withAnimation {
    currentDrawStatus = drawStatus
}
```

and you could check the draw status in the overlay like this:
```swift
if currentDrawStatus == .inProgress {
...
}
```

Reply:
@yo1995

Reply:
I'll add in this change.